migrations/helpers/schemeUtils.js

```diff
@@ -32,6 +39,28 @@ async function registerScheme ({
   console.log()
 }
 
-registerScheme.SchemePermissions = SchemePermissions
+async function setParameters ({
+  scheme,
+  parameters
+}) {
+  assert(scheme, `The parameter scheme was not defined`)
+  assert(parameters, `The parameter parameters was not defined`)
+  assert(typeof parameters === 'object', `The parameter should be an array`)
+  
+  console.log('\nSet scheme parameters:')
+  Object.keys(parameters).forEach(paramName => {
+    console.log(`  - ${paramName}: ${parameters[paramName]}`)
+  })
 
-module.exports = registerScheme
+  const paramList = Object.values(parameters)
```

Comment:
This requires fixed order of `parameters` object keys, so 
```
parameters: {
      voteParams: genesisProtocolParamsHash,
      intVote: genesisProtocolAddress,
      contractToCall: dutchXContractAddress
    }
```
can't be given in any other order, which is usually not what is expected when accepting an object input.
Better make `parameters` in to an array, imho.

Reply:
Very good feedback, thanks Dima. I will send a new commit soon to address this